app/models/order_detail.rb

```diff
@@ -771,6 +747,34 @@ def problem_order?
     problem
   end
 
+  def can_dispute?
+    in_review? && dispute_at.blank?
+  end
```

Comment:
I was comparing it to the old `can_dispute?` method, but I guess this is just to make all of them parallel and independent?